imports/plugins/core/accounts/client/components/adminInviteForm.js

```diff
@@ -67,30 +91,33 @@ class AdminInviteForm extends Component {
       );
     }
 
+    const matchingAccount = getUserByEmail(email);
+    const isEmailVerified = matchingAccount &&
+      matchingAccount.emails &&
+      matchingAccount.emails[0] &&
+      matchingAccount.emails[0].verified;
```

Comment:
Rather than using index 0, it would be more correct to find the item where `obj.address === email`